rsconnect/main.py

```diff
@@ -733,6 +735,91 @@ def deploy_notebook(
     ce.deploy_bundle().save_deployed_info().emit_task_log()
 
 
+# noinspection SpellCheckingInspection,DuplicatedCode
+@deploy.command(
+    name="voila",
+    short_help="Deploy Voila notebook to RStudio Connect [v2022.09.0+].",
```

Comment:
We should take a note (on our board) to revisit this version number, depending on when the feature lands.